openapi/openCDE-spec.yaml

```diff
@@ -313,4 +322,43 @@ components:
               items:
                 $ref: '#/components/schemas/DocumentReference'
           required: ['documentReferenceList']
-      required: ['_links', '_embedded']
\ No newline at end of file
+      required: ['_links', '_embedded']
+
+    DataType:
+      type: object
+      properties:
+        type:
+          type: string
+          enum:
+            - string
+            - boolean
+            - date-time
+            - integer
+            - number
+            - enum
+            - array
+        required:
+          type: boolean
+        enumValues:
+          type: array
+          items:
+            type: string
+        arrayType:
+          $ref: '#/components/schemas/DataType'
+      required: ['type', 'required']
+    
+    DataTypeWithName:
```

Comment:
I wonder if we should still leave this out from this commit, since it is not referred from anywhere.

Reply:
I agree. Just having the DataType would be good for now. Till we have a use for the other type.